ocaml/testsuite/tests/typing-layouts-err-msg/immediate64.ml

```diff
@@ -0,0 +1,16 @@
+(* TEST
+   flags = "-extension layouts_alpha"
+   * expect
+*)
+
+(*****************************)
+(* Immediate64 layout errors *)
+
+(* Local_mode_cross_check *)
+(* Only used within [Result.is_ok] and not thrown as an exception *)
+
+(* Gc_ignorable_check *)
+(* Only used within [Result.is_ok] and not thrown as an exception *)
+
+(* Separability_check *)
```

Comment:
Maybe this goes away or maybe not. But I see there's already a CR v2.8 nearby. Good. This is fine then. (Resolve upon reading.)